### ISSUE TITLE
“Skip to main content” button was unresponsive when tested with VoiceOver and Chrome.

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/platform-forms-client#7897]
From client: results of their form in fable testing:

> Ran 20 accessibility tests using Fable (10 in English and 10 in French). The accessibility tests mainly focused on screen reader compatibility, with a couple tests using screen magnification tools.

https://cds-snc.freshdesk.com/a/tickets/30088

**1 issue related to unresponsive “Skip to main content button”** (tester used Voice Over with Chrome). The button did not respond when they tried to select it.

> "On the landing page, the "skip to main content" link appears to do nothing."

**Relevant WCAG criteria: 2.1.1 Keyboard**

**Comment by thiessenp-cds** [comment]
See PR for a detailed explanation of the fix: #7934

**Comment by anikbrazeau** [comment]
**Update from Sep 21 stand-up**
> - PR pending review
> - Will flag to Tim when back
> - Bryan flagged it was turning hydration error, unresponsive to client
> - Pete to look into that

**Comment by Abi-Nada** [comment]
Throwing a hydration error (mismatch between Server HTML and client HTML javascript)

**Comment by thiessenp-cds** [comment]
Unable to reproduce the hydration error. Perhaps it was resolved in this fix.
